### ISSUE TITLE
Clarification on documentation?

### ISSUE THREAD

**Issue by amplicity** [EOA-Blockchain-Labs/ethereumonarm#30]
Amazing project. I've read through the documentation [here](https://ethereum.org/en/developers/tutorials/run-node-raspberry-pi/#consensus-layer) but it wasn't explicitly clear if i could run the consensus layer on mainnet? Is that possible?

**Comment by diglos** [comment]
Yes you can. Pick up Nimbus, Lighthouse or Prysm (Teku needs more resources as it is based on Java).

**Comment by amplicity** [comment]
Thank you, linking [here](https://ethereum.org/en/upgrades/get-involved/#clients) for anyone else who happens to stumble upon this issue.
